components/gravityforms/class-gf-convertkit.php

```diff
@@ -142,6 +142,7 @@ public function feed_settings_fields() {
 		);
 
 		$base_fields = $this->maybe_add_custom_field_mapping( $base_fields );
+		$base_fields = $this->add_tags_mapping( $base_fields );
```

Comment:
Should be called `maybe_add_tags_mapping()` to follow current syntax of `maybe_add_custom_field_mapping()`

Reply:
++